changelog.md

```diff
@@ -22,3 +22,4 @@
 
 ## TBD
 - Add `Vector::negative`
+- Add `Radians::from_degree`
```

Comment:
This is not the right function name

Reply:
Resolved in d6928345667f35b63543316836c535f1804fa94a